components/src/core/components/Input/Autocomplete/AutocompleteInput.vue

```diff
@@ -220,7 +230,11 @@ export default defineComponent({
         this.loading = false;
         this.dropdownOpen = false;
         this.pointer = -1;
-        this.$emit('update:modelValue', null);
+        this.searchTerm = null;
+        this.$emit(
+          'update:modelValue',
+          this.multiple ? this.selectedValues : this.searchTerm,
```

Comment:
I think this is incorrect. For multi-select option, it'll always return this.selectedValues

Reply:
this has been removed "this.selectedValues"